elastic/kibana
# [Response Ops] Fixing issue with empty row in execution log table when rule is running

Resolves https://github.com/elastic/kibana/issues/129126

## Summary

Updates the exec log aggregation to exclude `execute-start` events. This prevents an empty row from being returned for currently running rule executions.

## To Verify

See description in https://github.com/elastic/kibana/issues/129126 to reproduce the issue. With this branch, you should no longer see an empty row returned while a rule is mid-execution.


### Checklist
